lib/pbench/server/__init__.py

```diff
@@ -157,29 +155,27 @@ def _ref_datetime(self):
 
     @property
     def rest_uri(self):
-        return self._get_conf("pbench-server", "rest_uri")
+        return self.get_conf("pbench-server", "rest_uri")
 
     @property
     def max_retention_period(self) -> int:
-        """
-        Produce a timedelta representing the number of days the server allows
+        """Produce a timedelta representing the number of days the server allows
         a dataset to be retained.
 
         Returns
             delta time representing retention period
         """
         try:
             retention_days = int(
-                self._get_conf("pbench-server", "maximum-dataset-retention-days")
+                self.get_conf("pbench-server", "maximum-dataset-retention-days")
             )
         except (NoOptionError, NoSectionError):
             retention_days = self.MAXIMUM_RETENTION_DAYS
         return retention_days
 
-    def _get_conf(self, section, option):
-        """
-        _get_conf - get the option from the section, raising
-        BadConfig Error if it is empty, return the option value as a string.
+    def get_conf(self, section, option):
```

Comment:
So there are no callers of the old `get_conf` dependent on the behavior of returning `None` on error?

Reply:
I didn't see any....